### PR TITLE
Implement speakers screen

### PR DESCRIPTION
**Pull request by BKinya**
# Scope

_Please make sure to read https://github.com/droidconKE/droidconKE2022Android/docs/CONTRIBUTING.md
and check that you understand and have followed it as best as possible.

Explain what your feature does in a short paragraph.
- Implemented speakers screen

_ please check the below boxes
- [x] I have followed the coding conventions
- [ ] I have added/updated necessary tests
- [x] I have tested the changes added on a physical device
- [x] I have run `./codeAnalysis.sh` to make sure all lint/formatting checks have been done.

## Closes/Fixes Issues
_Declare any issues by typing `fixes #1` or `closes #1` for example so that the automation can kick
in when this is merged_

## Other testing QA Notes
_What have you tested specifically and what possible impacts/areas there are that may need retesting
by others._

Screenshots
### Phone
speakers

### Foldable
foldable

#### Tablet
tablet

### CODE REVIEW

**Review by wangerekaharun — APPROVED** [review]
Looks good, thanks for adding even screenshots for foldables and  tablet

**Comment by BKinya** [comment]
> Looks good, thanks for adding even screenshots for foldables and tablet


I'm honored! 🙏

File: presentation/src/main/java/com/android254/presentation/speakers/view/SpeakerComponent.kt [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+import coil.compose.AsyncImage
+import coil.request.ImageRequest
+import com.android254.presentation.R
+import com.android254.presentation.common.theme.DroidconKE2022Theme
+import com.android254.presentation.models.Speaker
+
+@Composable
+fun SpeakerComponent(
+    modifier: Modifier = Modifier,
+    speaker: Speaker = Speaker(name = "ABC", bio = "A quick brown fox ...")
```
Comment (chepsi) [inline presentation/src/main/java/com/android254/presentation/speakers/view/SpeakerComponent.kt:53]: `Speaker(name = "ABC", bio = "A quick brown fox ...")` -> remove this?

File: presentation/src/main/java/com/android254/presentation/common/navigation/Navigation.kt [hunk 2]
```diff
@@ -76,5 +78,10 @@ fun Navigation(
                 }
             )
         }
+        composable(Screens.Speakers.route) {
+            upDataAppBarState(false)
+            upDateBottomBarState(false)
+            SpeakersScreen(speakers = speakersList)
```
Comment (chepsi) [inline presentation/src/main/java/com/android254/presentation/common/navigation/Navigation.kt:84]: Please fetch the speaker list from the SpeakerScreenViewmodel.

File: presentation/build.gradle.kts [hunk 3]
```diff
@@ -89,6 +89,9 @@ dependencies {
     testImplementation(libs.compose.ui.test.junit)
     testImplementation(libs.android.test.espresso)
     testImplementation(libs.test.navigation)
+
+    // TODO: Find out how dependencies are being added
+    implementation("io.coil-kt:coil-compose:2.2.0")
```
Comment (chepsi) [inline presentation/build.gradle.kts]: check out `[ReadMe.MD](https://github.com/droidconKE/droidconKE2022Android#readme)` under dependencies bit.

Comment (BKinya) [reply presentation/build.gradle.kts]: Forgot to remove this 🤦‍♀️

**Comment by BKinya** [comment]
@chepsi I've made the improvements.

**Comment by michaelbukachi** [comment]
Hey @BKinya Please add some unit tests for this.

**Comment by BKinya** [comment]
> Hey @BKinya Please add some unit tests for this.

Hi @michaelbukachi, 

Sure, I'll add.

**Comment by BKinya** [comment]
> Hey @BKinya Please add some unit tests for this.

@michaelbukachi I have added a test. 😀

**Pull request closed without merging**